libs/sdk-common/src/utils/rest_client.rs

```diff
@@ -40,7 +40,7 @@ pub async fn get_and_log_response(
     let response = get_reqwest_client()?.get(url).send().await?;
     let status = response.status();
     let raw_body = response.text().await?;
-    debug!("Received response, status: {status}, raw response body: {raw_body}");
+    trace!("Received response, status: {status}, raw response body: {raw_body}");
```

Comment:
perhaps log the status only on debug? It is very helpful to understand if the chain service failed or not.

Reply:
Made the response status a debug log and the raw body a trace log